etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/AddEtorProcessingTag.java

```diff
@@ -9,8 +9,8 @@
 import java.util.Map;
 import org.hl7.fhir.r4.model.Bundle;
 
-/** Custom transformation to add an ETOR tag to the message header of a FHIR bundle. */
-public class addEtorProcessingTag implements CustomFhirTransformation {
+/** Adds the ETOR tag to a message header meta tag for RS processing. */
+public class AddEtorProcessingTag implements CustomFhirTransformation {
```

Comment:
Should the `transform` method have a `try`/`catch` and thrown a `RuleExecutionException` like the newer transformations?  And then unit tests for this too.

Reply:
Same